nasa/harmony
# HARMONY-1432: As a harmony service provider, I want to ensure there is exactly one UMM-S record for each harmony service chain.

## Jira Issue ID
HARMONY-1432

## Description
As a harmony service provider, I want to ensure there is exactly one UMM-S record for each harmony service chain so that it is simpler to configure harmony service related information

## Local Test Steps
1. Change a service chain configuration in services.yml to without `umm_s` configuration, verify harmony prints out a proper error message similar to `Error: There must be one and only one umm_s record configured as a string for harmony service: podaac/l2-subsetter` and harmony service does not start up.
2. Change a service chain configuration in services.yml to `umm_s` configured as a list, verify verify harmony prints out a proper error message similar to `Error: There must be one and only one umm_s record configured as a string for harmony service: podaac/l2-subsetter` and harmony service does not start up.

## PR Acceptance Checklist
